### PR TITLE
Fedora 22 and 23 support

### PR DESCRIPTION
**Pull request by aranc23**
I added boxes to the Vagrantfile, some logic to the get_facts.sh script, and the facts files generated by those changes for Fedora 22 and 23.

### CODE REVIEW

File: facts/Vagrantfile [hunk 1]
```diff
@@ -104,6 +104,20 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     host.vm.provision "shell", path: "get_facts.sh"
     host.vm.provision "shell", inline: "/sbin/shutdown -h now"
   end
+  config.vm.define "fedora-22-x86_64" do |host|
+    host.vm.box = "bento/fedora-22"
+    host.vm.provision "shell", inline: "dnf -y install redhat-lsb-core gcc glibc-devel"
```
Comment (mcanevet) [inline facts/Vagrantfile]: please do not install lsb packages as it generates facts that are not present by defaut on rehat oses

File: facts/Vagrantfile [hunk 2]
```diff
@@ -104,6 +104,20 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
...
+    host.vm.provision "file", source: "Gemfile", destination: "Gemfile"
+    host.vm.provision "shell", path: "get_facts.sh"
+    host.vm.provision "shell", inline: "/sbin/shutdown -h now"
+  end
+  config.vm.define "fedora-23-x86_64" do |host|
+    host.vm.box = "bento/fedora-23"
+    host.vm.provision "shell", inline: "dnf -y install redhat-lsb-core gcc glibc-devel"
```
Comment (mcanevet) [inline facts/Vagrantfile]: same here

**Comment by aranc23** [comment]
I removed redhat-lsb-core package from provisioning, updated the facts files, and updated the README.md to include Fedora 22/23 as supported.

**Comment by mcanevet** [comment]
@aranc23 thanks

**Pull request merged**